### DISCUSSION TITLE
Push latest release to Jenkins plugin center

### DISCUSSION THREAD

**Discussion by SunBlack** [jenkinsci/gitlab-plugin#1104] [General]
The plugin center of Jenkins currently [contains just 1.5.13](https://plugins.jenkins.io/gitlab-plugin/) during here already 1.5.17 is avaiable. Is it possible to push the latest release to the center, as it contains some fixes we need?

**Comment by pychamps** [comment]
Could you please release the latest version to jenkins plugin center.
we are randomly getting this issue below in our pipelines , which has seemed to be fixed in the latest version.
https://github.com/jenkinsci/gitlab-plugin/issues/895

**Comment by Stikus** [comment]
@markyjackson-taulia Any updates?

**Comment by sadok-f** [comment]
Any update on this one?

**Comment by irqus** [comment]
Could you please provide an update regarding this?
We have a huge issue with latest version available from Jenkins Plugin center `1.5.13`, we need to use the latest stable one.
@markyjackson-taulia could give us an update? 
Thank you.

**Comment by rajata07** [comment]
Any Update on this? Thanks!

**Comment by markjacksonfishing** [comment]
All,
Due to family medical constraints, I have started the following discussion as I can no longer maintain this plugin: https://github.com/jenkinsci/gitlab-plugin/discussions/1098

**Reply by sadok-f** [reply]
Thank you Marky for the clarification, all the best with your family.
